tests/acceptance/blocks/verticalresults.js

```diff
@@ -66,6 +69,19 @@ class VerticalResults {
     return this._selector.exists;
   }
 
+  /**
+   * wait for results to return based on search complete state 
+   * (default selector timeout of 10 seconds)
+   */
+   async waitOnSearchComplete () {
+    if(!this._preSearch.exists) {
```

Comment:
For my own edification, what is `preSearch`?

Reply:
it's one of the search states from SDK (presearch, loading, completed). I originally thought initial search on page may go from presearch to completed, skipping loading state. Turns out it's just testCafe unable to catch the loading transition. Updated to track query request/responses instead!